src/main/resources/application.conf

```diff
@@ -44,4 +44,5 @@ team {
   # { username: "username", gitlab-id: 111 }
   usernames = [
   ]
+  amount = 2
```

Comment:
Could you please add a comment with a description for the field

Reply:
done